release-team/role-handbooks/ci-signal/README.md

```diff
@@ -81,10 +87,11 @@ Here are some good early deliverables from the CI Signal lead between start of t
   - Assign the issue to appropriate SIG using /sig label
   - If you are aware of the individual associated with the enhancement area or issue, @mention of individual(s) and SIG leads tends to result in faster turn around
   - Add @kubernetes/sig-foo-test-failures to draw SIG-foo’s attention to the issue
-  - CC the release manager and bug triage lead if the issue needs attention
+  - CC @ci-signal github team on the issue to let rest of the team know about it, you might also CC the release lead and bug triage lead if the issue needs extra attention
     immediately
```

Comment:
Looks like this may have snuck down to a new line :)

Reply:
you have a good eye :)

Reply:
It actually originally was on newline, I don't think that was intentional :)